worlds/witness/data/static_logic.py

```diff
@@ -103,19 +104,31 @@ def read_logic_file(self, lines) -> None:
                 "Laser Hedges",
                 "Laser Pressure Plates",
             }
-            is_vault_or_video = "Vault" in entity_name or "Video" in entity_name
+            is_vault_or_video = "Vault" in entity_name
```

Comment:
if this no longer checks for video, the variably should probably be renamed

Reply:
True